src/main/java/com/jmb/springfactory/dao/rol/RolMongoServiceImpl.java

```diff
@@ -26,14 +33,24 @@
   
   @Override
   public Rol save(Rol rol) {
+    
+    // Retrieve permission from BD to get hibernete proxy session
+    val permissions = rol.getPermissions().stream()
+      .map(permission -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)